shell/config/router/navigation-guards/load-initial-settings.js

```diff
@@ -0,0 +1,13 @@
+import { fetchInitialSettings } from '@shell/utils/settings';
+
+export function install(router, context) {
+  router.beforeEach((from, to, next) => loadInitialSettings(from, to, next));
+}
+
+export async function loadInitialSettings(from, to, next) {
+  try {
+    await fetchInitialSettings();
+  } catch (ex) {}
```

Comment:
The original implementation executed fetchInitialSettings() with a try catch which swallowed and ignored the exception. I think we should consider fixing when we have time to dig deeper.